README.md

```diff
@@ -100,6 +100,21 @@ _Note: SwiftGen needs Xcode 8.3 to build, so installing via Homebrew requires yo
 
 ---
 </details>
+<details>
+<summary>Via <strong>Mint</strong></summary>
+
+To install SwiftGen via [Mint](https://github.com/yonaskolb/Mint), simply use:
+
+Note: Will work for Releases > 5.3.0 only.
+
+```sh
+$ brew install libxml2
```

Comment:
For what it's worth I didn't need this on my machine. Still not sure about the whole libxml2 story. Maybe you can add: `# may be needed`

Reply:
I had to do it (with a relatively new brew installation on the machine).

Reply:
Maybe add a comment that it might be needed for some/most people?